From 2c642ee15b6b9ce4fb8599b956f23b729d259bf8 Mon Sep 17 00:00:00 2001
From: Dinolek <18579460+Dinolek@users.noreply.github.com>
Date: Sat, 26 Oct 2019 22:49:17 +0200
Subject: [PATCH] incoming calls fix

---
 rilproxy/libril/ril_service.cpp | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/rilproxy/libril/ril_service.cpp b/rilproxy/libril/ril_service.cpp
index 16c3726f..9a70adeb 100644
--- a/rilproxy/libril/ril_service.cpp
+++ b/rilproxy/libril/ril_service.cpp
@@ -11771,8 +11771,14 @@ int radio::incomingCallIndicationInd(int slotId, int indicationType,
         Return<void> retStatus = radioService[slotId]->mRadioIndicationMtk->incomingCallIndication(
                 convertIntToRadioIndicationType(indicationType), inCallNotify);
         radioService[slotId]->checkReturnStatus(retStatus, RADIO_MTK);
+    } else if (radioService[slotId] != NULL && radioService[slotId]->mRadioIndication != NULL) {
+        if (response != NULL && responseLen == 7 * sizeof(char *)) {
+            char **strings = (char **) response;
+            dispatchInts(token, slotId, RIL_REQUEST_SET_CALL_INDICATION, 3, std::stoi(strings[3]),
+                         std::stoi(strings[0]), std::stoi(strings[4]));
+        }
     } else {
-        RLOGE("incomingCallIndicationInd: radioService[%d]->mRadioIndicationMtk == NULL", slotId);
+        RLOGE("incomingCallIndicationInd: radioService[%d]->mRadioIndication == NULL", slotId);
     }
 
 
-- 
2.23.0

